core/services/relay/evm/chain_reader.go

```diff
@@ -3,9 +3,12 @@ package evm
 import (
 	"context"
 	"fmt"
+	"reflect"
 	"strings"
 
 	"github.com/ethereum/go-ethereum/accounts/abi"
+	"github.com/google/uuid"
```

Comment:
linter fails this file with `File is not `goimports`-ed with -local github.com/smartcontractkit/chainlink`